tests/test_io.py

```diff
@@ -130,12 +166,11 @@ def test_column_series_irregular_columns(self):
         s = pe.Sheet(content, name_columns_by_row=0)
 
         pe.save_as(array=s, dest_file_name=self.testfile)
-        f = open(self.testfile, "r")
-        written_content = f.read()
-        f.close()
+
+        self._check_test_file('column_series_irregular_columns')
 
         # FIXME: numerical align lost when one cell is missing
```

Comment:
What's the impact of the [irregular columns](https://github.com/pyexcel/pyexcel-text/blob/master/tests/test_io.py#L85)? I believe that's what tabulate gives.